docs/source/usage/Ellipsoid.rst

```diff
@@ -0,0 +1,117 @@
+.. _transformation:
+
+================================================================================
+Ellipsoid  transformation
+================================================================================
+A ellipsoid is the mathematically defined surface which approximates the geiod, surface of the earth's gravity field, which is approximately the same as mean sea level.
+
+
+.. figure:: ./images/General_ellpsoid.png
+   :width: 500 px
+   :align: center
+   :alt:   Global and local fitting of the ellipsoid
+
+   proj-string: ``+proj=merc``
+
+
+
+A complete ellipsoid definition comprises a size (primary) and a shape (secondary)
+parameter.
+
+Ellipsoid Size parameters 
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+.. _ellipsoid_parameters : PROJ/docs/source/operations/options/
```

Comment:
It's fine to label the section so that it can be linked to from elsewhere in the docs but the labels need to be unique. Also, the path after it doesn't really make any sense in this context. What did you try to achieve with it?

```suggestion
.. _ellipsoid_size_parameters:
```